### PR TITLE
Add wasm compat data

### PR DESCRIPTION
**Pull request by Elchi3**
Adds compat data for https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly and subpages.

### CODE REVIEW

File: javascript/builtins/WebAssembly.json [hunk 1]
```diff
@@ -0,0 +1,1350 @@
...
+              "version_added": "57"
+            },
+            "chrome": {
+              "version_added": "57"
+            },
+            "chrome_android": {
+              "version_added": "57"
+            },
+            "edge": {
+              "version_added": "16"
```
Comment (chrisdavidmills) [inline javascript/builtins/WebAssembly.json]: I've got this listed as being available in 15 behind a flag. Where did you get 16 from? I did quite a bit of searching, and couldn't find any such info ;-)

File: javascript/builtins/WebAssembly.json [hunk 2]
```diff
@@ -0,0 +1,1350 @@
...
+            },
+            "edge_mobile": {
+              "version_added": true
```
Comment (chrisdavidmills) [inline javascript/builtins/WebAssembly.json]: I've got this listed as no on the existing compat tables, but their platform status page has it listed as a preview in Edge mobile (https://developer.microsoft.com/en-us/microsoft-edge/platform/status/webassemblymvp/). Doesn't this mean it's preffed off though?

File: javascript/builtins/WebAssembly.json [hunk 3]
```diff
@@ -0,0 +1,1350 @@
...
+            "status": {
+              "experimental": false,
+              "standard_track": true,
+              "deprecated": false
+            }
+          }
+        },
+        "Table": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Table",
```
Comment (chrisdavidmills) [inline javascript/builtins/WebAssembly.json]: Does Table also need an entry for 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Table/prototype

?

File: javascript/builtins/WebAssembly.json (hunk 1 above)
Comment (Elchi3) [reply javascript/builtins/WebAssembly.json]: I saw https://twitter.com/MSEdgeDev/status/894607665334177792 and caniuse is saying 16 now, too.

File: javascript/builtins/WebAssembly.json (hunk 2 above)
Comment (Elchi3) [reply javascript/builtins/WebAssembly.json]: Yeah, good point, I think we should indicate mobile support behind a pref still. Will update.

**Comment by Elchi3** [comment]
PTAL :)

**Comment by chrisdavidmills** [comment]
Looks good to me. r+

**Pull request merged**